README.md

```diff
@@ -34,51 +35,63 @@ git add HistFitter && git commit -m "Add HistFitter submodule"
 
 Recommended Root version is 6.28/00. This version of HistFitter is not compatible with ROOT versions < 6.28.
 
-## Setup
-
-The environment can be set doing
-
-```
-source setup.sh
-```
-
 An LCG release with the correct ROOT and Python versions can be loaded with
 
 ```
 source setup_lcg.sh
 ```
 
-
 ## Build
 
+Make your build directory and specify install directory. You have to point cmake to where the cloned source directory is located.
 ```
-cd src
-make
-cd ..
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=/path/to/install/ /path/to/histfitter/
+make -j4 install
 ```
-or with cmake:
+
+HistFitter was originally built and used in the same folder as the source code. If you prefer this option, simply go to the source folder and make the build directory there. This will treat the source code folder `histfitter` as the install location, creating the /lib folder directly there.
 ```
-cmake -S src -B lib -DCMAKE_BUILD_TYPE=Release
-cmake --build lib --clean-first --parallel 8
+cd histfitter
+mkdir build
+cd build
+cmake  ..
+make install
 ```
+This will create a lib folder with the installed libraries.
+
+
+## Setup
 
-## Directory structure
+The environment must be set by activating the setup script, located in the /install folder, in the directory where you want to work. This will create three new folders in your work folder: /config /data and /results. It also copies the `HistFactorySchema.dtd` file over to the config folder. Most importantly, the setup script sets the environment variable paths which are required for the scripts to work. The paths are set during install, and if any folders are moved, the build and install process must be rerun to update the setup script.
+
+```
+source /path/to/install/histfitter_setup.sh
```

Comment:
This path doesn't seem to be correct.  I think it should be .../install/bin/histfitter_setup.sh